src/promptflow/promptflow/_sdk/_service/app.py

```diff
@@ -137,9 +137,11 @@ def log_before_request_info():
             else:
                 request_body = request.get_data()
 
+            app.logger.info(
+                "Request coming in: %s, last request time: %s", request.url, app.config["last_request_time"]
```

Comment:
what's last request time? do we need it in log message, I think the logging itself will print some timestamp?

Reply:
last request time is used to record last http request time of flask app

Reply:
I understand that. But put it in this log line will be confusing.